apps/site/src/components/canvas/View.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import { forwardRef, useImperativeHandle, useRef } from "react";
+import { OrbitControls, View as ViewImpl } from "@react-three/drei";
+import { Three } from "@/helpers/components/Three";
+
+// export const Common = ({ color }: any) => (
+// 	<Suspense fallback={null}>
+// 		{color && <color attach="background" args={[color]} />}
+// 		<ambientLight intensity={0.5} />
+// 		<pointLight position={[20, 30, 10]} intensity={1} />
+// 		<pointLight position={[-10, -10, -10]} color="blue" />
+// 		<PerspectiveCamera makeDefault fov={40} position={[0, 0, 6]} />
+// 	</Suspense>
+// );
```

Comment:
Question: is there a reason why this is commented out?

Reply:
I left it there for reference. I'll take it out.